kms/kms.rb

```diff
@@ -174,18 +174,90 @@
       Cloudkms::CryptoKey.new(purpose: "ENCRYPT_DECRYPT")
   )
 
-  puts "Created version #{crypto_key_version.name} for key " +
-       "#{crypto_key} in key ring #{key_ring_id}"
+  version = crypto_key_version.name.split("/").last
```

Comment:
1. What do the other samples do?
2. Is there another way to get just the version number?
3. How important is it for us to print just the number?  Will users understand if crypto_key_version.name is the full path to the version?  Is that problematic?

I liked it how it was 😛